Makefile

```diff
@@ -307,29 +288,6 @@ gen-api-docs:
 	go install k8s.io/kops/cmd/kops-server
 	${GOPATH}/bin/apiserver-boot build docs --disable-delegated-auth=false --output-dir docs/apireference --server kops-server
 
-.PHONY: push
-# Will always push a linux-based build up to the server
-push: crossbuild-nodeup
-	scp -C .build/dist/linux/amd64/nodeup  ${TARGET}:/tmp/
-
-.PHONY: push-gce-dry
-push-gce-dry: push
-	ssh ${TARGET} sudo SKIP_PACKAGE_UPDATE=1 /tmp/nodeup --conf=metadata://gce/config --dryrun --v=8
-
-.PHONY: push-gce-dry
-push-aws-dry: push
-	ssh ${TARGET} sudo SKIP_PACKAGE_UPDATE=1 /tmp/nodeup --conf=/var/cache/kubernetes-install/kube_env.yaml --dryrun --v=8
-
-.PHONY: push-gce-run
-push-gce-run: push
-	ssh ${TARGET} sudo cp /tmp/nodeup /home/kubernetes/bin/nodeup
-	ssh ${TARGET} sudo SKIP_PACKAGE_UPDATE=1 /home/kubernetes/bin/nodeup --conf=/var/lib/toolbox/kubernetes-install/kube_env.yaml --v=8
-
-# -t is for CentOS http://unix.stackexchange.com/questions/122616/why-do-i-need-a-tty-to-run-sudo-if-i-can-sudo-without-a-password
-.PHONY: push-aws-run
-push-aws-run: push
```

Comment:
This is a helper method, will work.  It can be moved into a hack script if we want, but I hate to remove the functionality.